### PR TITLE
TELCODOCS#1451: Added forceWipeDisksAndDestroyAllData parameter

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s): 4.15

Issue: [TELCODOCS-1451](https://issues.redhat.com/browse/TELCODOCS-1451)

Link to docs preview:
https://69065--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-reference-file_logical-volume-manager-storage

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/lvms-creating-logical-volume-manager-cluster.adoc [hunk 1]
```diff
@@ -57,11 +57,12 @@ spec:
           optionalPaths:
           - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
           - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
+          forceWipeDisksAndDestroyAllData: true <5>
```
Comment (suleymanakbas91) [inline modules/lvms-creating-logical-volume-manager-cluster.adoc:60]:
```suggestion
          forceWipeDevicesAndDestroyAllData: true <5>
```
The name of the field is `forceWipeDevicesAndDestroyAllData`. I also updated the descriptions of the stories. I should've done it earlier. Sorry for the confusion!

File: modules/lvms-reference-file.adoc [hunk 2]
```diff
@@ -39,36 +39,38 @@ spec:
         optionalPaths:
         - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
         - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
-      thinPoolConfig: <6>
-        name: thin-pool-1 <7>
-        sizePercent: 90 <8>
-        overprovisionRatio: 10 <9>
+        forceWipeDisksAndDestroyAllData: true <6>
```
Comment (suleymanakbas91) [inline modules/lvms-reference-file.adoc:42]:
```suggestion
        forceWipeDevicesAndDestroyAllData: true <6>
```

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: modules/lvms-creating-logical-volume-manager-cluster.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/lvms-creating-logical-volume-manager-cluster.adoc]: Updated the parameter name.

File: modules/lvms-reference-file.adoc (hunk 2 above)
Comment (sr1kar99) [reply modules/lvms-reference-file.adoc]: Updated the parameter name.

**Comment by sr1kar99** [comment]
@suleymanakbas91 
Updated the parameter name as suggested. Could you please take a look.
Thanks.

**Comment by sr1kar99** [comment]
@mffiedler 
Could you please review the PR?
Thanks.

**Comment by radeore** [comment]
@mmakwana30 Hi Minal, could you please help review?

**Review by skrthomas — COMMENTED** [review]
One small comment. LGTM otherwise.

File: modules/lvms-creating-logical-volume-manager-cluster.adoc [hunk 3]
```diff
@@ -97,6 +99,17 @@ The `LVMCluster` resource creates the following system-managed CRs:
 `LVMVolumeGroup`:: Tracks individual volume groups across multiple nodes.
 `LVMVolumeGroupNodeStatus`:: Tracks the status of the volume groups on a node.
 
+[NOTE]
+====
+Wiping the disk can lead to inconsistencies in data integrity if any of the following conditions is met:
```
[the remark this answers is not in the document]
Comment (sr1kar99) [reply modules/lvms-creating-logical-volume-manager-cluster.adoc:104]: Hi @skrthomas 
Thank you for the review. Just one query on this comment - 

Even if only one of the three conditions is true, wiping the disk can lead to inconsistencies. 
It is not mandatory for all the conditions to be true. 

As "any" can be used can be used both in singular and plural, in this case where only one condition is expected to be met, which one would be appropriate - "is" or "are"?

[the remark this answers is not in the document]
Comment (sr1kar99) [reply modules/lvms-creating-logical-volume-manager-cluster.adoc:104]: @skrthomas Thanks for the confirmation.

**Review by sheriff-rh — APPROVED** [review]
LGTM, merging.

**Pull request merged**